src/models/ollama.py

```diff
@@ -0,0 +1,66 @@
+import os
+import ollama
+from tqdm.contrib.concurrent import thread_map
+
+from src.models.llm import LLM
+from src.utils.mylogger import MyLogger
+
+_model_name_map = {
+    "ollama-qwen-coder": "qwen2.5-coder:latest",
+    "ollama-qwen": "qwen2.5:32b",
+    "ollama-llama3": "llama3.2:latest",
+    "ollama-deepseek-32b": "deepseek-r1:32b",
+    "ollama-deepseek-7b": "deepseek-r1:latest",
+}
+_OLLAMA_DEFAULT_OPTIONS = {
+    "temperature": 0,
```

Comment:
Set these keys to the default values [here](https://github.com/ollama/ollama/blob/main/docs/modelfile.md#valid-parameters-and-values)

ex - temperature:0.8, num_predict: -1